# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1457398897 18000
#      Mon Mar 07 20:01:37 2016 -0500
# Node ID 1562e7e27f6dc1052d8ee3577b307e9f9d40951c
# Parent  3fb2bdea47a563ee86c72f03b2b0fac7e9dc605d
provide Make rule for recreating generated configuration shell scripts

* Makefile.am: New pattern rule for $(GEN_CONFIG_SHELL) files.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -70,16 +70,19 @@ GEN_CONFIG_SHELL = \
   build-aux/mk-mxarray-h.sh \
   build-aux/mk-version-h.sh \
   build-aux/subst-config-vals.sh \
   build-aux/subst-cross-config-vals.sh \
   build-aux/subst-default-vals.sh \
   build-aux/subst-f77-isnan-macro.sh \
   build-aux/subst-script-vals.sh
 
+$(GEN_CONFIG_SHELL) : %.sh : %.in.sh config.status
+	$(AM_V_GEN)$(SHELL) config.status $@-tmp $@
+
 AUTOCONF_SUBST_VARS = @AUTOCONF_SUBST_VARS@
 ALL_SUBST_VARS = \
   $(AUTOCONF_SUBST_VARS) \
   abs_top_builddir \
   abs_top_srcdir \
   api_version \
   version
 
